code/modules/economy/shippingmarket.dm

```diff
@@ -72,9 +71,6 @@
 				qdel(C)
 
 
-		SPAWN(300)
-			market_shift()
-
```

Comment:
hm
without this early market shift, when does the first mail crate actually arrive in a round? at the ~7 minute mark? ~14?

Reply:
It comes in at ~7 since the first market shift now happens at round-start